src/applications/auth/containers/AuthApp.jsx

```diff
@@ -156,44 +156,32 @@ export class AuthApp extends React.Component {
 
       // We're having trouble matching the user with MVI
       case '004':
-        header = 'We can’t match your information to our records';
+        header = 'Please try again later';
         alertContent = (
           <p>
-            We’re sorry. We can’t match the information you provided with what
-            we have in our Veteran records. We take your privacy seriously, and
-            we’re committed to protecting your information.
+            We’re sorry. Something went wrong on our end, and we couldn't sign
```

Comment:
Make sure these are using the curly apostrophes (`couldn't` -> `couldn’t`). `Cmd+Option+]`